utils/python/CIME/case_submit.py

```diff
@@ -33,14 +33,28 @@ def submit(caseroot, job=None, resubmit=None, no_batch=False, prereq_jobid=None)
 
     cimeroot = case.get_value("CIMEROOT")
     os.environ["CIMEROOT"] = cimeroot
+    # if case.submit is called with the no_batch flag then we assume that this 
+    # flag will stay in effect for the duration of the RESUBMITs
+    if resubmit is None:
```

Comment:
If resubmit is none then this is an initial job submission, if the no_batch flag is True set batch_system value to none
if it is not true set the batch_system value to whatever the default_value is.   If resubmit is true use the current value of batch_system.